snippets/react-snippets.code-snippets

```diff
@@ -49,6 +49,16 @@
         ],
         "description": "Add `role='feed'` to a dynamic list of articles. Add `aria-busy` if articles are being loaded or have been removed from the feed."
     },
+    "ApplicationAria":{
+        "prefix": [
+            "ApplicationAria",
+        ],
+        "body": [
+            "role='application'",
```

Comment:
we should add the associated attributes as well: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Application_Role#associated_wai-aria_roles_states_and_properties